docs/source/tutorial.rst

```diff
@@ -268,109 +268,7 @@ the runner (and every other robot on our team) move much more quickly.
 Take a screen recording of this whole process and send it to your software lead
 via Slack. Feel free to play around with any other params you see!
 
-5. Action Clients and building a position
```

Comment:
commit issue?

Reply:
Yeah, something happened, it should be there with this new commit.